languages/go/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -0,0 +1,3 @@
+# Hints
```

Comment:
If the document is empty, I'd remove it.

Reply:
updated!